src/core_plugins/elasticsearch/lib/__tests__/is_es_compatible_with_kibana.js

```diff
@@ -17,10 +17,6 @@ describe('plugins/elasticsearch', () => {
       it('when majors are equal, but ES minor is less than Kibana minor', () => {
         expect(isEsCompatibleWithKibana('1.0.0', '1.1.0')).to.be(false);
       });
-
-      it('when majors and minors are equal, but ES patch is less than Kibana patch', () => {
-        expect(isEsCompatibleWithKibana('1.1.0', '1.1.1')).to.be(false);
-      });
```

Comment:
Shouldn't we just change this assertion to be `true` and move it to the 'returns true' describe block?